packages/design-picker/src/hooks/use-theme-designs-query.ts

```diff
@@ -72,7 +72,7 @@ function apiThemeToDesign( { id, name, taxonomies, stylesheet }: any ): Design {
 		is_featured_picks: isFeaturedPicks,
 		showFirst: isFeaturedPicks,
 		...( STATIC_PREVIEWS.includes( id ) && { preview: 'static' } ),
-		design_type: is_premium ? 'premium' : 'standard',
```

Comment:
@valterlorran Is there any reason to remove the design_type? It made the metric `calypso_signup_design_type_submit` looks weird 😅

Reply:
I've proposed a [PR](https://github.com/Automattic/wp-calypso/pull/65227) to fix it

Reply:
Yes, it has made our statistics incorrect since June 30 :(
<img width="1255" alt="image" src="https://user-images.githubusercontent.com/5665959/177330533-825cbbd5-55dc-4010-b761-ab14997015dd.png">

Reply:
Oh 😞 , I think I accidentally removed it!  Sorry for the inconvenience and thanks for fixing it.

Reply:
No worries 🙂